src/CardPattern.res

```diff
@@ -100,4 +100,14 @@ let cardPatterns = [
     length: [16],
     pincodeRequired: false,
   },
+  {
+    issuer: "PagoBancomat",
+    pattern: %re(
+      "/^(10051\d{5}|10053\d{5}|10108\d{5}|10150\d{5}|10258\d{5}|10300\d{5}|10309\d{5}|20083\d{5}|20084\d{5}|20085\d{5}|20089\d{5}|30150\d{5}|30190\d{5}|30320\d{5}|30430\d{5}|30439\d{5}|30480\d{5}|30583\d{5}|30589\d{5}|30590\d{5}|30620\d{5}|30670\d{5}|30690\d{5}|30750\d{5}|30830\d{5}|30836\d{5}|30870\d{5}|31040\d{5}|31110\d{5}|31240\d{5}|31260\d{5}|31270\d{5}|31380\d{5}|31580\d{5}|31599\d{5}|31850\d{5}|32040\d{5}|32050\d{5}|32115\d{5}|32390\d{5}|32400\d{5}|32420\d{5}|32440\d{5}|32500\d{5}|32630\d{5}|32681\d{5}|32682\d{5}|32685\d{5}|32968\d{5}|33320\d{5}|33870\d{5}|33990\d{5}|34000\d{5}|34029\d{5}|34030\d{5}|34253\d{5}|34310\d{5}|34400\d{5}|34420\d{5}|34880\d{5}|34938\d{5}|35000\d{5}|35899\d{5}|35980\d{5}|35990\d{5})$/"
```

Comment:
can we put one comment what this regex is doing?

And please check in multiple browsers for this regex.